scylla/src/transport/session.rs

```diff
@@ -1040,6 +1041,7 @@ impl Session {
                 Statement {
                     token: self.calculate_token(ps, first_serialized_value)?,
                     keyspace: ps.get_keyspace_name(),
+                    is_confirmed_lwt: false,
```

Comment:
What if `ps` is a confirmed LWT statement? Perhaps it would make sense to forward this flag here.

Reply:
Please correct me if I'm wrong, but I think that we decided to postpone it until a PR which adds support for routing batches is merged. That PR was merged recently (https://github.com/scylladb/scylla-rust-driver/pull/573), so changing the highlighted line slightly should just work.

On the other hand, you should probably add tests for it, so it's not a matter of changing one line after all. If that's a large amount of work then it's OK to do in a followup.

Reply:
Let's do it in a followup.